### PR TITLE
Issue 597 - Fix tp6m summary graph link to more details

### PR DESCRIPTION
**Pull request by airimovici**
https://github.com/mozilla-frontend-infra/firefox-health-dashboard/issues/597

### CODE REVIEW

**Comment by airimovici** [comment]
Link for testing: 
https://deploy-preview-600--firefox-health-dashboard.netlify.com/android

File: src/android/TP6mAggregate.jsx [hunk 1]
```diff
@@ -342,7 +342,7 @@ class TP6mAggregate_ extends Component {
                           path: '/android/tp6m',
                           query: {
                             test,
-                            platform,
+                            platform: browserPlatform.id,
```
Comment (klahnakoski) [inline src/android/TP6mAggregate.jsx]: Writing `platform: browserPlatform.id` means something is not named correctly:  ES6 re/destructuring promotes using a single name across functions and modules for the same thing.  When you see a name change, either something has the wrong name, or logic is wrong.  I am concerned that `platform` is different from `browserPlatform.id`.   I am guessing the problem you are trying to solve is somewhere else in the code.

* is `platform` value correct?  maybe it must be changed to `"fenix-*"`?

Comment (klahnakoski) [reply src/android/TP6mAggregate.jsx]: @airimovici  a gentle reminder that this PR is still open.

**Comment by airimovici** [comment]
The problem here is that the nav expects a browser-platform id and currently it is given a browser-id. With this wrong value, it will always got ot the default value: 'fenix-g5'

What can be done to eliminate the confusion is to change the nav picker id from TP6m page from platform to browserPlatform.

File: src/android/TP6mAggregate.jsx (hunk 1 above)
Comment (airimovici) [reply src/android/TP6mAggregate.jsx]: There is a confusion between platform and browserPlatform in several components and parts of the code. For this case in particular I suggested a fix below: to eliminate the confusion we should change the nav picker id from TP6m page from `platform` to `browserPlatform`

**Comment by airimovici** [comment]
@klahnakoski I did the changes as I described above

**Pull request merged**